docs/breadcrumb/toc.yml

```
@@ -10,6 +10,9 @@ items:
           - name: .NET Aspire (Preview)
             tocHref: /dotnet/aspire/
             topicHref: /dotnet/aspire/index
+          - name: What's new in .NET Aspire
```

I might add "docs" here:

```suggestion
          - name: What's new in .NET Aspire Docs
```